corehq/apps/hqwebapp/static/hqwebapp/js/daterangepicker.config.js

```diff
@@ -37,8 +37,11 @@ $.fn.createDateRangePicker = function (
     };
     var hasStartAndEndDate = !_.isEmpty(startdate) && !_.isEmpty(enddate);
     if (hasStartAndEndDate) {
-        config.startDate = new Date(startdate);
-        config.endDate = new Date(enddate);
+        // Parse dates as local dates to avoid timezone conversion
+        var startParts = startdate.split('-');
```

Comment:
Could be worth a comment that this operates specifically on a "YYYY-MM-DD" date.